### ISSUE TITLE
Touch Bar issue: App Switcher widget causing erratic behaviour before crashing

### ISSUE THREAD

**Issue by MiracleBlue** [fifafu/BetterTouchTool#418]
#### Description of your bug/question/feature request...:
I have had BTT set up with a button group, under which I have an app switcher widget set up, that has worked perfectly for the week and a half that I have been using it.  Today, I went to use it and the app switcher seemed invisible, and other buttons in the group were in the wrong positions.  So I tried tweaking configurations for it, but after a few tries, BTT crashed completely with the following error (taken from the Apple Crash Reporter):

```
Application Specific Information:
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** setObjectForKey: key cannot be nil'
Performing @selector(triggerAction:) from sender NSButton 0x60000014f150
terminating with uncaught exception of type NSException
abort() called
```

Seems to be repeatable, too.  I can reopen BTT, go to the button group, and it will crash about half of the time when opening the button group, and the app switcher remains invisible and the other buttons are all misplaced.

**What kind of Mac are you using?**
MacBook Pro Late-2016 15" with Touch Bar

**Which input device (e.g. MacBook Trackad, Magic Mouse, Magic Trackpad Touch Bar) is affected by this?**
MacBook Touch Bar

**Which versions of macOS and BetterTouchTool are you running?**
Mac OS X 10.12.3
BTT 2.02 (598)

**Other information:** (e.g. stacktraces, related issues, screenshots,suggestions how to fix, etc)
Here is the crash file: [BetterTouchTool_2017-02-06-142722_crash.txt](https://github.com/fifafu/BetterTouchTool/files/753630/BetterTouchTool_2017-02-06-142722_crash.txt)

**Comment by fifafu** [comment]
Should be fixed in 2.021 alpha! Thanks for reporting :-)

**Comment by MiracleBlue** [comment]
Thank you, that seems to have fixed it!  That was super quick!  :D
